### PR TITLE
Release 2.8.0

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: docs/reference/replicated-cli.md [hunk 1]
```diff
@@ -187,7 +187,13 @@ replicated snapshot list app <app id>
 replicated snapshot list location <location>
 ```
 
-{{< version version="2.5.0" >}} Restore an application from the specified snapshot.  When optional `--dismiss-prechecks` flag is specified, failed preflight checks will be ignored.  The optional `--node-timeout` parameter indicates how long to wait for the initial node to connect.  The default is 60 seconds.
+{{< version version="2.8.0" >}} Start a snapshot. When the optional `--wait` flag is specified, the command will block until the current snapshot is complete. The optional `--exclude-app-data ` flag indicates to back up only Replicated data, excluding all vendor application data from the snapshot.
```
Comment (divolgin) [inline docs/reference/replicated-cli.md]: extra space in `--exclude-app-data `

**Review by divolgin — APPROVED** [review]

File: docs/packaging-an-application/docker-swarm.md [hunk 2]
```diff
@@ -26,6 +26,22 @@ Some of the standard Replicated features operate differently or are not supporte
 ### External Private Images
 External private images are not supported currently. Replicated hosts a [private registry](/getting-started/replicated-private-registry) that you can use to ship private images. Replicated also supports public (unauthenticated) images in any registry.
 
+### Airgapped Installations
+{{< version version="2.8.0" >}} Airgapped installations work as expected when running in swarm mode. All images included in your swarm application must be specified a new `images` section of your YAML in ordered to be included in the airgap bundle your customer will download. See below for an example.
```
Comment (wlaoh) [inline docs/packaging-an-application/docker-swarm.md]: > must be specified a new `images` section of your YAML

sounds like it should be

> must be specified in the new `images` section of your YAML

File: docs/packaging-an-application/template-functions.md [hunk 3]
```diff
@@ -663,24 +663,6 @@ PremkitNetworkName() string
 
 PremkitNetworkName returns the name of the premkit docker network.
 
-```yml
-services:
-  voting-app:
-    image: gaiadocker/example-voting-app-vote:good
-    environment:
-      REPLICATED_INTEGRATIONAPI: {{repl PremkitAPIAddress }}
```
Comment (wlaoh) [inline docs/packaging-an-application/template-functions.md:671]: Vendor will still need to add environment variables to access the Integration API, should we really remove this part as well?

File: docs/packaging-an-application/template-functions.md [hunk 4]
```diff
@@ -705,24 +687,6 @@ StatsdNetworkName() string
 
 StatsdNetworkName returns the name of the Statsd docker network.
 
-```yml
-services:
-  voting-app:
-    image: gaiadocker/example-voting-app-vote:good
-    environment:
-      REPLICATED_STATSD_ADDRESS: {{repl StatsdAddress }}
```
Comment (wlaoh) [inline docs/packaging-an-application/template-functions.md:713]: Vendor will still need to add environment variables to access the Statsd container should we really remove this part as well?

File: docs/distributing-an-application/add-nodes.md [hunk 5]
```diff
@@ -14,8 +14,11 @@ url        = "/docs/distributing-an-application/add-nodes"
 
 The instructions to add additional nodes are different depending the running scheduler.
 
-## Replicated Scheduler
+### Replicated Scheduler
 To add additional nodes when running on the Replicatd schdduler, refer to the instructions on the /cluster page of the Admin Console. For details, visit the [instructions for adding additional Replicated nodes](/distributing-an-application/add-nodes-replicated).
```
Comment (wlaoh) [inline docs/distributing-an-application/add-nodes.md]: Replicatd -> Replicated

File: docs/distributing-an-application/installing.md [hunk 6]
```diff
@@ -42,7 +42,7 @@ We provide a short set of steps to install Replicated manually, for situations w
 
 [Continue with a manual install](/distributing-an-application/installing-manually)
 
-### Airgap Install
+### Airgapped Install
 Some installations may not have outbound Internet access.  Replicated provides you the option to do airgapped installs to support those customers.  To install your customer will download Replicated, your license and the airgap installation file containing your application.
```
Comment (wlaoh) [inline docs/distributing-an-application/installing.md]: To install, your customer will download...

File: docs/packaging-an-application/clustering.md [hunk 7]
```diff
@@ -70,4 +62,8 @@ components:
       initial: 3
 ```
 
-In the example above we will require at least 2 hosts. We will start 3 instances of the freighter/worker container on each of these 2 hosts.
+In the example above at least 2 hosts are required. 3 instances of the freighter/worker container with be started on each of these 2 hosts.
```
Comment (wlaoh) [inline docs/packaging-an-application/clustering.md]: container with be started -> container will be started

Comment (marccampbell) [reply docs/packaging-an-application/clustering.md]: @wlaoh will you just make the edits directly to this PR?

**Pull request merged**